cryptostore/data/gd.py

```diff
@@ -141,6 +141,7 @@ def write(self, bucket: str, path: str, file_name: str, **kwargs):
 
         """
 
+        socket = StorageEngines['socket']
```

Comment:
socket is part of the standard library, so you can just import it normally, no need to use the storage engine

Reply:
@bmoscon 
Ok, done. Thanks.